docs/version-2-upgrade.md

```diff
@@ -0,0 +1,385 @@
+}
+
+module "kinesis_firehose_access" {
+  source  = "symopsio/kinesis-firehose-addon/aws"
+  version = "~> 1.1"
+
+  environment = var.environment
+  iam_role_name = module.runtime_connector.sym_runtime_connector_role.name
+}
+```
+
+Optionally, you may choose to use `moved` configuration blocks to migrate your Terraform state, instead of replacing the existing IAM policies with new ones:
+```terraform
+# The following blocks may be removed after applying the updated configuration
+moved {  
+  from = module.runtime_connector.module.aws_kinesis_firehose[0].aws_iam_policy.this
+  to   = module.kinesis_firehose_access.aws_iam_policy.this  
+}  
+  
+moved {  
+  from = module.runtime_connector.aws_iam_role_policy_attachment.aws_kinesis_firehose_attach[0]
+  to   = module.kinesis_firehose_access.aws_iam_role_policy_attachment.attach_firehose_access[0]  
+}
+```
+
+### Refactoring the `aws/kinesis-data-stream` Addon
+If `aws/kinesis-data-stream` is declared as an addon, update your configuration to add the `symopsio/kinesis-data-stream-addon/aws` module explicitly.
+
+For example, given this previous configuration:
+```terraform
+module "runtime_connector" {
+  source  = "symopsio/runtime-connector/aws"
+  version = "~> 1.0"
+
+  environment = var.environment
+  
+  addons       = ["aws/kinesis-data-stream"]
+  addon_params = {
+    "aws/kinesis-data-stream" = {
+       "stream_arns" = ["arn:aws:kinesis:*:111122223333:stream/my-stream"]
+    }
+  }
+}
+```
+
+An updated configuration would be:
+
+```terraform
+module "runtime_connector" {
+  source  = "symopsio/runtime-connector/aws"
+  version = "~> 2.0"
+
+  environment = var.environment
+}
+
+module "kinesis_data_stream_access" {
+  source  = "symopsio/kinesis-data-stream-addon/aws"
+  version = "~> 1.1"
+
+  environment   = var.environment
+  stream_arns   = ["arn:aws:kinesis:*:111122223333:stream/my-stream"]
+  iam_role_name = module.runtime_connector.sym_runtime_connector_role.name
+}
+```
+
+Optionally, you may choose to use `moved` configuration blocks to migrate your Terraform state, instead of replacing the existing IAM policies with new ones:
+```terraform
+# The following blocks may be removed after applying the updated configuration
+moved {  
+  from = module.runtime_connector.module.aws_kinesis_data_stream[0].aws_iam_policy.this
+  to   = module.kinesis_data_stream_access.aws_iam_policy.this  
+}  
+  
+moved {  
+  from = module.runtime_connector.aws_iam_role_policy_attachment.aws_kinesis_data_stream_attach[0]
+  to   = module.kinesis_data_stream_access.aws_iam_role_policy_attachment.attach_datastream_access[0]  
+}
+```
+
+## Removed Input: `custom_external_id`
+This input is no longer supported, and the `external_id` of the AWS IAM Role declared by this module will always be a random UUID.
+
+## Input `policy_arns` has been removed
+This input is no longer supported, and any additional AWS IAM Policies should be attached directly to the AWS IAM Role outputted by the module.
```

Comment:
```suggestion
This input is no longer supported, and any additional AWS IAM Policies should be attached directly to the AWS IAM Role output by the module.
```
"Outputted" has always read weird to me; though it is technically correct, I think using "output" is more common.

Reply:
oh good it wasn't just me; I waffled back and forth between `output` and `outputted` when writing this 😂